src/3rd_party/CMakeLists.txt

```diff
@@ -205,6 +205,11 @@ else()
   get_filename_component(EMHASHMAP_LIBS_DIRECTORY ${EMHASHMAP_LIB} DIRECTORY)
   set(BSON_LIBS_DIRECTORY ${BSON_LIBS_DIRECTORY} CACHE INTERNAL "Installation path of bson libraries" FORCE)
   set(EMHASHMAP_LIBS_DIRECTORY ${BSON_LIBS_DIRECTORY} CACHE INTERNAL "Installation path of emashmap libraries" FORCE)
+  add_custom_target(
+      libbson
+      DEPENDS ${BSON_LIBS_DIRECTORY}
+      DEPENDS ${EMHASHMAP_LIBS_DIRECTORY}
```

Comment:
@LuxoftAKutsan  Are these additions related to the policy changes? Can you please explain further?

Reply:
@mrapitis yep, this is not related to defect. It is just annoying issue that I was fixed in scope. 
I placed it in separate commit. So it can be safely removed.